src/transformers/models/hubert/__init__.py

```diff
@@ -21,6 +21,7 @@
 
 
 _import_structure = {
+    ".wav2vec2.feature_extraction_wav2vec2": ["Wav2Vec2FeatureExtractor"],
```

Comment:
IMO - this is ok. What do you think @sgugger ? In short we need to make HuBERT work with `AutoFeatureExtractor` and it uses the exact same feature extractor than Wav2Vec2. Either we import Wav2Vec2 here or we add a hack to how feature extractors are loaded in `models/auto/modeling_auto_feature_extractor.py` WDYT?

Reply:
This is less hacky than what we did with MT5. Works for me.